packages/babel-helper-plugin-utils/src/index.ts

```diff
@@ -30,6 +34,12 @@ export function declare<
   };
 }
 
+export function declarePreset<Option = {}, API = PresetAPI>(
+  builder: (api: API, options: Option, dirname: string) => PresetObject,
+): (api: API, options: Option, dirname: string) => PresetObject {
+  return declare(builder as any) as any;
+}
```

Comment:
Can we avoid a function call? Something like
```suggestion
export const declarePreset = declare as
  <Option = {}, API = PresetAPI>(
    builder: (api: API, options: Option, dirname: string) => PresetObject,
  ): (api: API, options: Option, dirname: string) => PresetObject;
```